### ISSUE TITLE
Few commands do not work for Roboteq HDC2460

### ISSUE THREAD

**Issue by sachinlodhi** [Miker2808/PyRoboteq#4]
so everything is fine and after following the tutorial I got both of these commands working and motors moving :

```python
controller.send_command(cmds.DUAL_DRIVE,50, 50)
controller.dual_motor_control(100, 100) # Send command to the controller
```

but following does not work:
```python
from PyRoboteq import RoboteqHandler
from PyRoboteq import roboteq_commands as cmds

controller.send_command(cmds.SET_SPEED, 1, 100) # send 'set speed' command to channel 1 (first argument) with the value of up to 1000 RPM (second argument)
```

It does nothing.

**Comment by Miker2808** [comment]
Hello, this library is simply community made bridge between python and the Roboteq controllers.
therefore it is strongly advisable to check the commands you are planning to use in the manual.

as for the command `SET_SPEED` you've tried. This command is sending the `!S` command to the controller.
From the manual you can see that: 
_"In the Closed-Loop Speed mode, this command will cause the motor to spin at the desired RPM speed. In Closed-Loop Position modes, this commands determines the speed
at which the motor will move from one position to the next. It will not actually start the
motion."_

in short, this command will only assign the RPM the motor will aim for when you are calling the drive commands, and will not actually spin the motors.
the `!G` is the command that actually tells the controller to move the motor. (which is `cmds.GO` in python)
